src/core/Akka.Streams.Tests/Dsl/QueueSourceSpec.cs

```diff
@@ -265,7 +265,7 @@ public void QueueSource_should_complete_watching_future_with_failure_if_material
         {
             this.AssertAllStagesStopped(() =>
             {
-                var tempMap = ActorMaterializer.Create(Sys);
+                var tempMap = ActorMaterializer.Create(Sys, ActorMaterializerSettings.Create(Sys)); // need to create a new materializer to be able to shutdown it
```

Comment:
One weakness of the `DefaultMaterializer` design is that it can techincally be shut down by any caller. We should probably harden this in the future such that it can only be shutdown when the `ActorSystem` terminates, but that'll require creating a new subclass of the `ActorMaterializer` to do that properly.